src/js/angular/repositories/ontop-repo.directive.js

```diff
@@ -17,6 +17,9 @@ function ontopRepoDirective($uibModal, RepositoriesRestService, toastr, Upload,
 
     function linkFunc($scope) {
 
+        // A link to all Ontop configuration keys
+        $scope.ontoProperiesLink = 'https://ontop-vkg.org/guide/advanced/configuration.html';
```

Comment:
Is this something static? Can it be changed?

Reply:
I exported it as a constant on top.